### PR TITLE
Replacing "local network" image by a `diff` example

### PR DESCRIPTION
**Pull request by pedrorezende**
This PR replaces the **Spinning up a local network** image by an example using `diff` block, wrapped by <Expandable /> component. 

Also sticks the position of <Expandable /> in order to work better with long blocks of code.

### CODE REVIEW

File: packages/docs/next.config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const withNextra = require("nextra")({
   theme: "nextra-theme-docs",
   themeConfig: "./theme.config.tsx",
   latex: true,
+  defaultShowCopyCode: true,
```
Comment (karbyshev) [inline packages/docs/next.config.js:5]: I think we should do it in a separate PR.

Comment (bengtlofgren) [reply packages/docs/next.config.js:5]: Meh, I agree but it's fineee

**Review by bengtlofgren — APPROVED** [review]

File: packages/docs/next.config.js (hunk 1 above)
Comment (bengtlofgren) [reply packages/docs/next.config.js:5]: Actually I changed my mind, let's remove it for now and open a new PR

Comment (pedrorezende) [reply packages/docs/next.config.js]: Sorry about that! Thanks for fixing that @bengtlofgren
